README.md

```diff
@@ -74,7 +169,17 @@ func MyController(ctx context.Context, config *rest.Config) error {
 		return result, err
 	}))
 
-	return controllerFactory.Start(ctx, 5)
+	err = controllerFactory.Start(context.TODO(), 5)
```

Comment:
Could add a comment here about what `Start` is actually starting and what the number means

Reply:
Included this in the step but will comment here as well.